src/components/ToDoList/ToDoList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { todoListType } from '../../typedefs/todoListType';
+import { ToDoItem } from '../ToDoItem';
+import './ToDoList.scss';
+
+export const ToDoList = ({
+  visibleTodos,
+  onToggle,
+  onDelete,
+  onEdit,
+  handleKeyPress,
+  setEditedValue,
+}) => (
+  <ul className="todo-list">
+    {visibleTodos.map(todo => (
+      <ToDoItem
+        key={todo.id}
+        {...{ todo }}
```

Comment:
```suggestion
        {...todo}
```
зачем дополнительные скобки?

Reply:
если не делать дополнительные скобки, то массив высыпается в объект и на входе в доч. компонент выдаёт undefiend, я уже пробовал по разному, получается 
либо так: {...{ todo }} 
либо так: todo={todo}
подскажи, если знаешь ещё вариант